docs/docs-content/clusters/public-cloud/azure/required-permissions.md

```diff
@@ -0,0 +1,366 @@
+    | Microsoft.Network/loadBalancers/backendAddressPools/join/action | Subscription |
+    | Microsoft.Network/loadBalancers/delete | Subscription |
+    | Microsoft.Network/loadBalancers/inboundNatRules/delete | Subscription|
+    | Microsoft.Network/loadBalancers/inboundNatRules/join/action | Subscription |
+    | Microsoft.Network/loadBalancers/inboundNatRules/read | Subscription |
+    | Microsoft.Network/loadBalancers/inboundNatRules/write | Subscription |
+    | Microsoft.Network/loadBalancers/read | Subscription |
+    | Microsoft.Network/loadBalancers/write | Subscription |
+    | Microsoft.Network/networkInterfaces/delete | Subscription |
+    | Microsoft.Network/networkInterfaces/join/action | Subscription |
+    | Microsoft.Network/networkInterfaces/read | Subscription |
+    | Microsoft.Network/networkInterfaces/write | Subscription |
+    | Microsoft.Network/networkSecurityGroups/read | Subscription |
+    | Microsoft.Network/networkSecurityGroups/securityRules/delete | Subscription |
+    | Microsoft.Network/networkSecurityGroups/securityRules/read | Subscription |
+    | Microsoft.Network/networkSecurityGroups/securityRules/write | Subscription |
+    | Microsoft.Network/privateDnsZones/A/delete | Subscription |
+    | Microsoft.Network/privateDnsZones/A/read | Subscriptionp |
+    | Microsoft.Network/privateDnsZones/A/write | Subscription |
+    | Microsoft.Network/privateDnsZones/delete | Subscription |
+    | Microsoft.Network/privateDnsZones/read | Subscription |
+    | Microsoft.Network/privateDnsZones/virtualNetworkLinks/delete | Subscription |
+    | Microsoft.Network/privateDnsZones/virtualNetworkLinks/read | Subscription |
+    | Microsoft.Network/privateDnsZones/virtualNetworkLinks/write | Subscription |
+    | Microsoft.Network/privateDnsZones/write | Subscription |
+    | Microsoft.Network/publicIPAddresses/delete | Subscription |
+    | Microsoft.Network/publicIPAddresses/join/action | Subscription |
+    | Microsoft.Network/publicIPAddresses/read | Subscription |
+    | Microsoft.Network/publicIPAddresses/write | Subscription |
+    | Microsoft.Network/routeTables/delete | Subscription |
+    | Microsoft.Network/routeTables/read | Subscription |
+    | Microsoft.Network/routeTables/write | Subscription |
+    | Microsoft.Network/virtualNetworks/join/action | Subscription |
+    | Microsoft.Resources/subscriptions/resourceGroups/read | Subscription |
+    | Microsoft.Network/virtualNetworks/read | Subscription |
+    | Microsoft.Network/virtualNetworks/subnets/join/action | Subscription |
+    | Microsoft.Network/virtualNetworks/subnets/read | Subscription |
+    | Microsoft.Compute/galleries/images/read | Subscription |
+    | Microsoft.Compute/galleries/images/versions/read| Subscription |
+
+
+
+</TabItem>
+
+
+</Tabs>
+
+
+
+
+
+
+
+
+## Dynamic Placement
+
+If you want to support dynamic placement use cases, ensure Palette has the following actions at the subscription level: 
```

Comment:
To stay consistent with the previous section, this needs an explanation for when users should choose dynamic placement.

As you've described under static placement, that would be "when you have a pre-existing network resource group, VNet, subnets and security groups you want Palette to use when deploying a cluster". What is the use case here?